duckduckgo/privacy-essentials-safari
# Deprecating harmful terminology

Task/Issue 1 URL: https://app.asana.com/0/1144193759412024/1180899597720227/f
Task/Issue 2 URL: https://app.asana.com/0/0/1181779178785307/f
Tech Design URL:
CC:

**Description**:
Visual and code changes deprecating harmful terminology

**Steps to test this PR**:
1. Open Dashboard
2. Make sure it contains these changes:
'Unprotected Sites' instead of 'Manage Whitelist', 
'Added to Unprotected Sites' instead of 'Added to Whitelist', 
'Removed from Unprotected Sites' instead of 'Removed from Whitelist'

3. Open the app
4. Make sure it contains these changes:
'Unprotected Sites' instead of 'Manage Whitelist', 
'No unprotected sites' instead of 'Empty Whitelist' when the list is empty
No 'whitelist' word in the description
'Uprotected sites' heading when the list has items

5. Make sure unprotected sites works as expected

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
